lib/resources/r4/condition.yaml

```diff
@@ -0,0 +1,119 @@
+---
+name: Condition
+field_name_base_url: http://hl7.org/fhir/r4/condition-definitions.html#Condition
+fields:
+- name: clinicalStatus
+  required: 'No'
+  cardinality: 0..1
+  type: CodeableConcept
+  description: active | inactive | resolved
```

Comment:
need to update to include all clinical statuses.

Reply:
Added. [diff](https://github.com/cerner/fhir.cerner.com/pull/273/commits/4872fd105329788054750c0c23ab3e1c4ec4459a#diff-abb99d2139d79f8a2a8f1d214f5f6882R9)